src/randgraphs.jl

```diff
@@ -50,6 +50,31 @@ function erdos_renyi(n::Integer, p::Real; is_directed=false)
     return g
 end
 
+
+using Distributions
+#faster for sparse erdos renyi graphs if p≈1, then use the erdos_renyi function
+function sparse_erdos_renyi(n::Integer, p::Real; is_directed=false)
+    if is_directed
+        possibleedges = n*(n - 1)
+        g = DiGraph(n)
+    else
+        possibleedges = (n^2-n)/2
+        g = Graph(n)
+    end
+    numedges = rand(Binomial(Int(possibleedges), p))
```

Comment:
Int() construction doesn't work in 0.3 - perhaps a version check here until 0.4 gets released?